rocAL/rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -0,0 +1,264 @@
+import amd.rocal.types as types
```

Comment:
I am trying this PR. Looks like ops.py is deleted from amd/rali and not added back in amd/rocal.
Please fix that